xscen/aggregate.py

```diff
@@ -761,7 +761,12 @@ def produce_horizon(
 
         if ds_sub is not None:
             # compute indicators
-            ind_dict = compute_indicators(ds=ds_sub, indicators=indicators)
+            ind_dict = compute_indicators(
+                ds=ds_sub.squeeze(dim="warminglevel")
```

Comment:
why? this should show with many wl, so a dim of 1 should also be fine, no ?

Reply:
We compute a single warming level at a time.

Without that change, I got this error when trying to compute the `doy_qmax` indicator:
```
INFO:xscen.indicators:1 - Computing JQ1MAXHP.
Traceback (most recent call last):
  File "/usr/local/pycharm-community-2022.2.3/plugins/python-ce/helpers/pydev/_pydevd_bundle/pydevd_exec2.py", line 3, in Exec
    exec(exp, global_vars, local_vars)
  File "<string>", line 1, in <module>
  File "/hidden/path/xscen/xscen/config.py", line 217, in _wrapper
    return func(*args, **kwargs)
  File "/hidden/path/xscen/xscen/indicators.py", line 143, in compute_indicators
    out = ind(ds=ds)
  File "/hidden/path/xclim/xclim/core/indicator.py", line 842, in __call__
    outs = self.compute(**compute_das, **kwargs, **var_kwargs)
  File "<boltons.funcutils.FunctionBuilder-187>", line 2, in select_resample_op
  File "/hidden/path/xclim/xclim/core/units.py", line 1148, in wrapper
    out = func(*args, **kwargs)
  File "/hidden/path/xclim/xclim/indices/generic.py", line 93, in select_resample_op
    return r.map(op)
  File "/hidden/path/xarray/core/resample.py", line 241, in map
    combined = super().map(func, shortcut=shortcut, args=args, **kwargs)
  File "/hidden/path/xarray/core/groupby.py", line 1382, in map
    return self._combine(applied, shortcut=shortcut)
  File "/hidden/path/xarray/core/groupby.py", line 1401, in _combine
    applied_example, applied = peek_at(applied)
  File "/hidden/paths/xarray/core/utils.py", line 188, in peek_at
    peek = next(gen)
  File "/hidden/path/xarray/core/groupby.py", line 1381, in <genexpr>
    applied = (maybe_wrap_array(arr, func(arr, *args, **kwargs)) for arr in grouped)
  File "/hidden/path/xclim/xclim/indices/generic.py", line 99, in doymax
    out = da.time.dt.dayofyear.isel(time=i, drop=True)
  File "/hidden/path/xarray/core/dataarray.py", line 1413, in isel
    ds = self._to_temp_dataset()._isel_fancy(
  File "/hidden/path/xarray/core/dataset.py", line 2704, in _isel_fancy
    new_var = var.isel(indexers=var_indexers)
  File "/hidden/path/xarray/core/variable.py", line 1378, in isel
    return self[key]
  File "/hidden/path/xarray/core/variable.py", line 899, in __getitem__
    dims, indexer, new_order = self._broadcast_indexes(key)
  File "/hidden/path/xarray/core/variable.py", line 745, in _broadcast_indexes
    return self._broadcast_indexes_vectorized(key)
  File "/hidden/path/xarray/core/variable.py", line 884, in _broadcast_indexes_vectorized
    return out_dims, VectorizedIndexer(tuple(out_key)), new_order
  File "/hidden/path/xarray/core/indexing.py", line 420, in __init__
    raise ValueError(
ValueError: Vectorized indexing with Dask arrays is not supported. Please pass a numpy array by calling ``.compute``. See https://github.com/dask/dask/issues/8958.
```